HAL9000:
  current_table: 0